frontend/src/App.css

```diff
@@ -23,49 +30,134 @@
 }
 
 /*Page Home*/
-h2 {
+
+.Home {
+  display: flex;
+  flex-direction: column;
+  justify-content: center;
+  align-items: center;
+}
+
+.Home h2 {
+  text-align: center;
+  color: #ffffff;
+  padding: 2rem;
+  font-weight: 100;
+}
+
+.Home img {
+  width: 10rem;
+}
+
+.buttonRegister,
+.buttonLogin {
+  border: 2px solid #dec439;
+  background-color: #dec439;
+  color: #010707;
+  text-decoration: none;
+  text-align: center;
+  padding: 0.5rem;
+  margin: 0.2rem;
+  border-radius: 2rem;
+  font-size: 1.2rem;
+  font-weight: 800;
+  width: 8rem;
+}
+
+/*Page Register*/
+
+.PresInscip {
   text-align: center;
+  margin-bottom: 1rem;
+}
+
+.PresInscip h1 {
   color: #dec439;
-  padding: 5rem;
-  margin-top: 6rem;
 }
 
-/*Page SignIn*/
+.Inscription label {
+  display: block;
+  margin-bottom: 0.2rem;
+}
 
-.bouton {
+.Inscription {
+  color: #fff;
   display: flex;
-  align-items: center;
   flex-direction: column;
+  justify-content: center;
+  min-height: 90.5vh;
```

Comment:
90.5 is a strange value: remove fixed values, and use flex-grow (according you're in a flex container)